README.md

```diff
@@ -1,17 +1,102 @@
 ## WFD_import
-A batch import of European water data based on the Water Framework Directive
-reporting.
+*WFD import* are a collection of scripts and tools for batch importing of
+European water data, reported under the Water Framework Directive, to Wikidata.
 
-To run it you will have to install `wikidata-stuff` using:
-`pip install git+https://github.com/lokal-profil/wikidata-stuff.git`
+It currently supports harvesting data on River Basin Districts (RBDs) and
+Surface Water Bodies (SWBs).
 
-*Note*: You might have to add the `--process-dependency-links` flag to the above
-command if you are running a different version of pywikibot from the required one.
+### How to run
+To do a basic run simply call either `RBD.py` or `swb_import.py` from the
+commandline together with the `-in_file:<path>` argument. The `-in_file` <path>
+value should be the url to an RBDSUCA .xml file for a RBD import or an SWB
+file(there is one per RBD) for an SWB import.
 
-### Note
-This repo was split of from 
-[lokal-profil/wikidata_batches](https://github.com/lokal-profil/wikidata_batches)
-so the history might be a bit mixed up.
+The xml files only include the names of the RBD/SWB in English (if any), to
+load the names in a local language use the `-gml_file:<path>` argument together
+with the path to a .gml file belonging to the RBD or SWB dataset.
+
+You can also use the `WfdBot.load_data()` to create a json dump of either of
+the .xml or .gml files. The local json dumps can then be used together with the
+`-in_file` or `gml_file` argument.
+
+To create new items, rather than just enriching pre-existing ones, use the
+`-new` flag. A new item is one where the unique RBD/SWB code is not yet
+associated with an item on Wikidata.
+
+To ensure no data is written to Wikidata you may use the `-simulate` flag. The
+`-preview_file:<path>` also prevents data from being written to Wikidata,
```

Comment:
Changed to one per paragraph for each. Sticking them at the start would however require a substantial restructuring. For users who want it that way they can just run the script with the `-help` flag. (added a not about all common pywikibot flags being supported).